Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -30,6 +32,7 @@ public class FocusProvider : BaseCoreSystem, IMixedRealityFocusProvider
         private readonly Dictionary<uint, IMixedRealityPointerMediator> pointerMediators = new Dictionary<uint, IMixedRealityPointerMediator>();
         private PointerHitResult hitResult3d = new PointerHitResult();
         private PointerHitResult hitResultUi = new PointerHitResult();
+        private int maxQuerySceneResults = 64;
```

Comment:
@julenka, do you have any data on some of the more complex scenes that people were building? 64 seems like it could be a small number potentially depending on how many near interaction things that exist.

Reply:
Ah, I guess it's part of the profile anyway, so it's changeable at least there.

Reply:
I should probably set this to 0 to make it clear that this value gets blown away by the profile value. I have it set to 128 in the profile by default.